packages/statistics/index.rst

```diff
@@ -0,0 +1,849 @@
+the `T statistic <https://en.wikipedia.org/wiki/Student%27s_t-test>`_,
+and the `p-value <https://en.wikipedia.org/wiki/P-value>`_ (see the
+function's help)::
+
+    >>> stats.ttest_1samp(data['VIQ'], 0)   # doctest: +ELLIPSIS
+    (...30.088099970..., 1.32891964...e-28)
+
+.. tip::
+   
+    With a p-value of 10^-28 we can claim that the population mean for
+    the IQ (VIQ measure) is not 0.
+
+2-sample t-test: testing for difference across populations
+...........................................................
+
+We have seen above that the mean VIQ in the male and female populations
+were different. To test if this is significant, we do a 2-sample t-test
+with :func:`scipy.stats.ttest_ind`::
+
+    >>> female_viq = data[data['Gender'] == 'Female']['VIQ']
+    >>> male_viq = data[data['Gender'] == 'Male']['VIQ']
+    >>> stats.ttest_ind(female_viq, male_viq)   # doctest: +ELLIPSIS
+    (...-0.77261617232..., 0.4445287677858...)
+
+Paired tests: repeated measurements on the same indivuals
+----------------------------------------------------------
+
+.. image:: auto_examples/images/plot_paired_boxplots_1.png
+   :target: auto_examples/plot_pandas.html
+   :scale: 70
+   :align: right
+
+PIQ, VIQ, and FSIQ give 3 measures of IQ. Let us test if FISQ and PIQ are
+significantly different. We can use a 2 sample test::
+
+    >>> stats.ttest_ind(data['FSIQ'], data['PIQ'])   # doctest: +ELLIPSIS
+    (...0.46563759638..., 0.64277250...)
+
+The problem with this approach is that it forgets that there are links
+between observations: FSIQ and PIQ are measured on the same individuals.
+Thus the variance due to inter-subject variability is confounding, and
+can be removed, using a "paired test", or `"repeated measures test"
+<https://en.wikipedia.org/wiki/Repeated_measures_design>`_::
+
+    >>> stats.ttest_rel(data['FSIQ'], data['PIQ'])   # doctest: +ELLIPSIS
+    (...1.784201940..., 0.082172638183...)
+
+.. image:: auto_examples/images/plot_paired_boxplots_2.png
+   :target: auto_examples/plot_pandas.html
+   :scale: 60
+   :align: right
+
+This is equivalent to a 1-sample test on the difference::
+
+    >>> stats.ttest_1samp(data['FSIQ'] - data['PIQ'], 0)   # doctest: +ELLIPSIS
+    (...1.784201940..., 0.082172638...)
+
+|
+
+T-tests assume Gaussian errors. We
+can use a `Wilcoxon signed-rank test
+<http://en.wikipedia.org/wiki/Wilcoxon_signed-rank_test>`_, that relaxes
+this assumption::
+
+    >>> stats.wilcoxon(data['FSIQ'], data['PIQ'])   # doctest: +ELLIPSIS
+    (274.5, 0.106594927...)
+
+.. note::
+
+   The corresponding test in the non paired case is the `Mann–Whitney U
+   test <http://en.wikipedia.org/wiki/Mann%E2%80%93Whitney_U>`_,
+   :func:`scipy.stats.mannwhitneyu`.
+
+.. topic:: **Exercice**
+   :class: green
+
+   * Test the difference between weights in males and females.
+
+   * Use non parametric statistics to test the difference between VIQ in
+     males and females.
+
+|
```

Comment:
Some conclusions here? If I understand well, comparing VIQs of male and female shows that one cannot reject that the two quantities have the same mean (p-value of 0.44..), but it might help the reader to write it in so many words.